tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib

```diff
@@ -97,7 +97,7 @@ function setup_testenv #[dtst]
 #
 function cleanup_testenv
 {
-	if is_global_zone && ismounted "$TESTDIR1" "ufs" ; then
+	if is_global_zone && ! ismounted "$TESTDIR1" "$NEWFS_DEFAULT_FS" ; then
```

Comment:
I believe you have a stray `!` here.  We want to cleanup the test ext2 filesystems on cleanup if they're mounted.